dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphIntegrationTest.java

```diff
@@ -94,10 +94,10 @@ public void testJaxen() throws RepositoryException {
     DependencyGraph graph = DependencyGraphBuilder.getCompleteDependencies(jaxen);
 
     List<Update> updates = graph.findUpdates();
-    Truth.assertThat(updates).hasSize(5);
+    Truth.assertThat(updates).hasSize(3);
```

Comment:
assertion changes because the dependency graph has changed. It has less number of nodes.